azure-pipelines.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  branches:
+    include:
+    - master
+name: $(Date:yyyyMMdd)$(Rev:-rr)
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: master
+jobs:
+- job: Job_1
+  displayName: Build and test
+  pool:
+    vmImage: windows-latest
+  steps:
+  - checkout: self
+
+  - task: PowerShell@2
+    displayName: Get Version from Directory.Build.props
+    inputs:
+      targetType: inline
+      script: >-
+        $xml = [Xml] (Get-Content .\src\Directory.Build.props)
+
+        $version = [String] $xml.Project.PropertyGroup.Version
+
+        $version = $version.Trim()
+
+        Write-Host "##vso[task.setvariable variable=Version;]$version"
+
+  - task: UseDotNet@2
+    displayName: Use .NET Core 5.0.x
+    inputs:
+      version: 5.0.x
```

Comment:
Change to 6

Reply:
doing a step-by-step approach.  Here it will just be 5, but once this gets merged, I will update on the other PR to be net 6